multimodal-search-tirg/app.py

```diff
@@ -31,24 +31,17 @@ def config():
     os.environ['JINA_PORT'] = '45678'
 
 def print_result(resp):
-    print('Works')
-    # for d in resp.search.docs:
-    #     vi = d.uri
-    #     result_html.append(f'<tr><td><img src="{vi}"/></td><td>')
-    #     for kk in d.matches:
-    #         kmi = kk.uri
-    #         result_html.append(f'<img src="{kmi}" style="opacity:{kk.score.value}"/>')
-    #         # k['score']['explained'] = json.loads(kk.score.explained)
-    #     result_html.append('</td></tr>\n')
+    import io
+    from base64 import b64decode
+    from PIL import Image
+    image_data_uri = resp.search.docs[0].matches[0].uri
```

Comment:
can u paste some screenshots of results?

Reply:
Result image popup

![image](https://user-images.githubusercontent.com/23080576/98098775-c7cd5000-1eb4-11eb-9bfb-475e6403e8b8.png)

Reply:
Bo will merge the executor code into the executor module and then we can start right?

Reply:
Once Bo does a PR with his code to this branch, we can merge it and u can take over and start rounding the proper executors with the tests. 

But first from a `test side` there is still some stuff to do:
- Test with real code of executors (no copy paste to a notebook)
- Test with batches
- Save some of the tests to add them to unit tests of executors.

These tasks can also be taken by you if Bo has no time and would block